dashboard/db/schema.rb

```diff
@@ -537,4 +551,7 @@
   add_index "workshops", ["name"], name: "index_workshops_on_name", using: :btree
   add_index "workshops", ["program_type"], name: "index_workshops_on_program_type", using: :btree
 
+  add_foreign_key "hint_view_requests", "levels"
+  add_foreign_key "hint_view_requests", "scripts"
+  add_foreign_key "hint_view_requests", "users"
```

Comment:
Hmm. I notice that there are no other foreign_key declarations in here; these lines were automatically added. Should they not have been?